test/jdk/java/nio/file/Files/probeContentType/Basic.java

```diff
@@ -187,6 +187,7 @@ public static void main(String[] args) throws IOException {
                 new ExType("xls", List.of("application/vnd.ms-excel")),
                 new ExType("xlsx", List.of("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")),
                 new ExType("7z", List.of("application/x-7z-compressed")),
+                new ExType("wasm", List.of("application/wasm")),
```

Comment:
Could you please also add  8297609 to the `@bug` list on line 25?

Reply:
@dfuch  Done.